### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by nacm1269**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8">
+  <title>Nathan Muran</title>
+  <link type="text/css" rel="stylesheet" href="styles.css">
```
Comment (charliecog) [inline index.html]: always link your stylesheet last so that you can override styles from other locations

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
```
Comment (charliecog) [inline styles.css]: auto not required

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+/* HEADER SECTION */
+header {
+  display: flex;
```
Comment (charliecog) [inline styles.css:12]: not doing anything, remove

File: index.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  <script src="https://kit.fontawesome.com/c6e95da7df.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+
+  <!-- HEADER AND LINKS TO DIFFERENT SECTIONS -->
+
+  <header class="header-section">
+    <section>
```
Comment (charliecog) [inline index.html]: section tag not needed, remove

File: index.html [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  <!-- FONT AWESOME -->
+  <script src="https://kit.fontawesome.com/c6e95da7df.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+
+  <!-- HEADER AND LINKS TO DIFFERENT SECTIONS -->
+
+  <header class="header-section">
```
Comment (charliecog) [inline index.html]: this is the only header tag on the page so the class is unecessary, remove and just use header as the css selector

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+  margin: 0;
+  padding: 10px 10px 0px;
+}
+
+header h2 {
+  color: #A9EEFF;
+  font-family: 'Raleway', sans-serif;
+  font-size: 1.8rem;
+  margin: 0;
+  padding: 0px 10px 600px 10px;
```
Comment (charliecog) [inline styles.css]: 600px padding seems a little over the top. Perhaps setting a defined height on the parent header element and using flex to position these bits would be preferable

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+  color: #ffffff;
+  font-family: 'Satisfy', cursive;
+  font-size: 3.2rem;
+  margin: 0;
+  padding: 10px 10px 0px;
+}
+
+header h2 {
+  color: #A9EEFF;
+  font-family: 'Raleway', sans-serif;
```
Comment (charliecog) [inline styles.css]: Just set this in the body tag and remove everywhere else

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+  margin: 0 auto;
+  padding: 75px 75px;
+}
+
+.about-me-text {
+  padding: 25px 0;
+}
+
+.about-me-icons {
+  align-items: center;
```
Comment (charliecog) [inline styles.css]: not doing anything, remove

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+  display: flex;
+  font-size: 4rem;
+  flex-direction: row;
```
Comment (charliecog) [inline styles.css]: not doing anything, remove

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+.contact-links a:hover {
+  color: #76B3BF;
+}
+
+.email-address {
+  text-decoration: none;
+}
+
+.copyright-text {
+  display: flex;
```
Comment (charliecog) [inline styles.css]: get rid of display flex and justify content center and replace with text-align: center

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+  width: 32%;
+}
+
+.portfolio article img {
+  width: 100%;
+}
+
+/* .project-article {
+  width: 32%;
+  margin: 10px;
```
Comment (charliecog) [inline styles.css]: removed commented out code

**Review by charliecog — COMMENTED** [review]
Looking good! couple of bits then your there!

File: styles.css (hunk 6 above)
Comment (nacm1269) [reply styles.css]: Added height to the header element. Used flex-direction: column and added margin-top: auto to .nav-links to reposition the nav links without the padding

**Review by charliecog — APPROVED** [review]
looks great

**Pull request merged**